src/plone_sphinx_theme/assets/styles/content/_images.scss

```diff
@@ -3,3 +3,7 @@ img.inline {
   margin: 0;
   height: 1.2em;
 }
+
+.reference.external::after {
```

Comment:
you may want to check with has pseudo selector if an a tag is in there.
https://developer.mozilla.org/en-US/docs/Web/CSS/:has